### PR TITLE
Update simulator link

### PR DESCRIPTION
**Pull request by PeterJCLaw**
While we left a redirect in place it seems that that only works for the path without a trailing slash. This change updates the url to the new one and removes the trailing slash so that it should continue to work even if the url changes again.

### CODE REVIEW

**Review by antoinepetty — APPROVED** [review]

**Review by antoinepetty — APPROVED** [review]

**Comment by antoinepetty** [comment]
Doesn't seem to like my approval - I am not worthy

**Comment by PeterJCLaw** [comment]
@RealOrangeOne do you know why @antoinepetty can't approve this/

**Comment by RealOrangeOne** [comment]
I don't have access to see the permissions setup, but my assumption would be that it's set up so that volunteers not in a "team" don't have write access to the repo (required for an approval to matter)

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**
